qa/CMakeLists.txt

```diff
@@ -5,5 +5,5 @@ endif()
 
 if(WITH_TESTS)
   include(AddCephTest)
-  add_tox_test(qa TOX_ENVS py3 flake8 import-tasks)
+  add_tox_test(qa TOX_ENVS py3 flake8 import-tasks mypy)
```

Comment:
```
mypy run-test: commands[0] | mypy .
workunits/fs/misc/filelock_interrupt.py:26: error: Module has no attribute "F_OFD_GETLK"; maybe "F_GETLK"?
workunits/fs/misc/filelock_interrupt.py:27: error: Module has no attribute "F_OFD_SETLK"; maybe "F_SETLK"?
workunits/fs/misc/filelock_interrupt.py:28: error: Module has no attribute "F_OFD_SETLKW"; maybe "F_SETLKW"?
Found 3 errors in 1 file (checked 232 source files)
```

probably could allow following patch
```diff
diff --git a/qa/workunits/fs/misc/filelock_interrupt.py b/qa/workunits/fs/misc/filelock_interrupt.py
index 5ebc480ba8e..b261d74fbc0 100755
--- a/qa/workunits/fs/misc/filelock_interrupt.py
+++ b/qa/workunits/fs/misc/filelock_interrupt.py
@@ -23,9 +23,9 @@ def timeout(seconds):
 """
 introduced by Linux 3.15
 """
-fcntl.F_OFD_GETLK = 36
-fcntl.F_OFD_SETLK = 37
-fcntl.F_OFD_SETLKW = 38
+setattr(fcntl, "F_OFD_GETLK", 36)
+setattr(fcntl, "F_OFD_SETLK", 37)
+setattr(fcntl, "F_OFD_SETLKW", 38)
 
 
 def main():
```

Reply:
Fascinating. I did run `tox -e mypy` locally without seeing this error.

Reply:
fixed.